pennylane_ionq/device.py

```diff
@@ -231,7 +231,7 @@ def _submit_job(self):
         # state (as a base-10 integer string) to the probability
         # as a floating point value between 0 and 1.
         # e.g., {"0": 0.413, "9": 0.111, "17": 0.476}
-        self.histogram = job.data.value["histogram"]
+        self.histogram = job.data.value

```

Comment:
is the job data value now the histogram?

Reply:
Yes, it used to be that one of the returned fields in the result of querying a task was `{..., data: {'histogram': {states and probs here}, 'registers': None}, ...}`. Now data is retrieved from the `results` url, and it only includes `{states and probs here}`.